linux_os/guide/system/network/network-nftables/nftables_rules_permanent/bash/sle15.sh

```diff
@@ -0,0 +1,18 @@
+# platform = sle15
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+{{% set nftables_family_names = ['bridge', 'arp', 'inet'] %}}
+
+{{{ bash_instantiate_variables("var_nftable_master_config_file") }}}
+
+if [! -f "${var_nftable_master_config_file}"]; then
```

Comment:
code style

Reply:
:+1:  Should have all the needed spaces in  d7a048a